assets/component-search.css

```diff
@@ -1,32 +1,70 @@
 .search__input.field__input {
```

Comment:
When tabbing the outline is cut off at the bottom at these 2 places:

https://screenshot.click/06-01-yjs3x-tc9kq.png

https://screenshot.click/06-01-3n5yb-rul2p.png